test/martha/smoketest_v2.test.js

```diff
@@ -6,7 +6,8 @@
  */
```

Comment:
We would still need to pass the `BASE_URL` here while running the smoke tests right?

Reply:
Yes, the `BASE_URL` is needed for smoke testing.